config.w32

```diff
@@ -224,6 +224,52 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no" && mongoc_ssl_found) {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
+
+    PHP_MONGODB_CFLAGS += "\
+      /I" + configure_module_dirname + "/src/libmongocrypt/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt/kms-message/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt-compat \
+    ";
+
+    var mongocrypt_opts = {
+      MONGOCRYPT_ENABLE_TRACE: 1,
+      MONGOCRYPT_IS_POSIX: 0,
+      MONGOCRYPT_IS_WIN: 1,
+      MONGOCRYPT_ENABLE_CRYPTO: 0,
+      MONGOCRYPT_ENABLE_CRYPTO_LIBCRYPTO: 0,
+      // TODO: Support building with Secure Channel on Windows
+      MONGOCRYPT_ENABLE_CRYPTO_CNG: 0,
+      // Secure Transport does not apply to Windows
+      MONGOCRYPT_ENABLE_CRYPTO_COMMON_CRYPTO: 0
+    };
+
+    // Generated with: find src/libmongocrypt/src -name '*.c' -print0 | cut -sz -d / -f 4- | sort -dz | tr '\000' ' '
+    var PHP_MONGODB_MONGOCRYPT_SOURCES="crypto/cng.c crypto/commoncrypto.c crypto/libcrypto.c crypto/none.c mongocrypt-binary.c mongocrypt-buffer.c mongocrypt.c mongocrypt-cache.c mongocrypt-cache-collinfo.c mongocrypt-cache-key.c mongocrypt-ciphertext.c mongocrypt-crypto.c mongocrypt-ctx.c mongocrypt-ctx-datakey.c mongocrypt-ctx-decrypt.c mongocrypt-ctx-encrypt.c mongocrypt-key-broker.c mongocrypt-key.c mongocrypt-kms-ctx.c mongocrypt-log.c mongocrypt-marking.c mongocrypt-opts.c mongocrypt-status.c mongocrypt-traverse-util.c os_posix/os_mutex.c os_posix/os_once.c os_win/os_mutex.c os_win/os_once.c";
```

Comment:
I forgot that `ADD_SOURCES` doesn't like paths in the list of source files. This apparently isn't a problem for Autotools, but it causes linking problems on Windows. Since libmongocrypt differs from libmongoc in that it nests source code within directories parallel to other sources, we may need to modify the `find` command here to not recurse.

I'm inclined to just use `-maxdepth 1` instead of pursuing a [POSIX-compliant solution](https://unix.stackexchange.com/questions/275637/limit-posix-find-to-specific-depth), since it's macOS-friendly and is likely to work on both of our development machines.

Reply:
Ah, I didn't know that. Since I'm still unable to compile on Windows I didn't catch that. Using `-maxdepth 1` for `find` works fine on my machine, I used this previously when I excluded the crypto sources to conditionally include them depending on the crypto library chosen.